security/pkg/k8s/chiron/utils.go

```diff
@@ -64,6 +65,67 @@ func getRandomCsrName(secretName, namespace string) string {
 	return name
 }
 
+func SignWorkloadCert(certClient certclient.CertificateSigningRequestInterface, csr string, identities []string) ([]byte, []byte, error) {
```

Comment:
Pretty sure we could use a single method for signing both Istiod DNS and workload certs.